app/assets/stylesheets/common/_core.scss

```diff
@@ -146,7 +146,7 @@ input[type='radio'] {
 }
 
 .help-message {
-  @include border-radius(5px);
+  border-radius: $radius;
```

Comment:
Properties should be ordered background, border, border-radius, display, padding, position, top, width, z-index